e2e/utils/create-project-utils.ts

```diff
@@ -57,27 +58,29 @@ export function newProject({
       }
 
       // TODO(jack): we should tag the projects (e.g. tags: ['package']) and filter from that rather than hard-code packages.
-      const packages = [
-        `@nrwl/angular`,
-        `@nrwl/eslint-plugin-nx`,
-        `@nrwl/express`,
-        `@nrwl/esbuild`,
-        `@nrwl/jest`,
-        `@nrwl/js`,
-        `@nrwl/linter`,
-        `@nrwl/nest`,
-        `@nrwl/next`,
-        `@nrwl/node`,
-        `@nrwl/nx-plugin`,
-        `@nrwl/rollup`,
-        `@nrwl/react`,
-        `@nrwl/storybook`,
-        `@nrwl/vite`,
-        `@nrwl/web`,
-        `@nrwl/webpack`,
-        `@nrwl/react-native`,
-        `@nrwl/expo`,
-      ];
+      const packages = packagesToInstall?.length
+        ? packagesToInstall
```

Comment:
This is not so simple. We use a single cache per machine which would lead to all other E2E projects running on that same machine after storybook ending up with only `angular` and `storybook`.

Can you revert this change, please?

Reply:
But let's find a way to not have to install ALL packages, or maybe improve the `runCreateWorkspace` function!